src/utils/text.js

```diff
@@ -32,7 +32,7 @@ function formatKeyShortcut(shortcut: string): string {
   }
   return shortcut.replace(
     /CommandOrControl\+|CmdOrCtrl\+/g,
```

Comment:
I think if you want to remove the `+` you just remove the two `\+` in this line.